src/pages/management/configuration-api/v3.5/index.mdx

```diff
@@ -2057,10 +2057,12 @@ Updates a property value within a license. This operation **doesn't** overwrite
 |                      |                                                                                   |
 | -------------------- | --------------------------------------------------------------------------------- |
 | Method URL           | `https://api.livechatinc.com/v3.5/configuration/action/update_license_properties` |
-| Required scopes__*__ | -                                                                                 |
+| Required scopes      | `properties.license.value--my:ro`*, `properties.license.value--all:ro`**          |
```

Comment:
are you sure about `ro` here? This method updates properties

Reply:
Replaced with `rw`.